src/sentry/constants.py

```diff
@@ -695,18 +695,14 @@ def from_str(cls, string: str) -> int:
 # Also it covers: livez, readyz
 HEALTH_CHECK_GLOBS = [
     "*healthcheck*",
-    "*healthy*",
-    "live",
-    "live[z/-]*",
-    "*[/-]live",
-    "*[/-]live[z/-]*",
-    "ready",
-    "ready[z/-]*",
-    "*[/-]ready",
-    "*[/-]ready[z/-]*",
     "*heartbeat*",
     "*/health",
+    "*/healthy",
```

Comment:
This would still match a transaction like `GET api/food/snacks/healthy` though, right?

Reply:
Yeah it's still flaky. Our goal is to try to fix it briefly and then possibly start efforts to make it a user-definable setting.